docs/advanced-backtesting.md

```diff
@@ -46,29 +46,25 @@ ranging from the simplest (0) to the most detailed per pair, per buy and per sel
 
 More options are available by running with the `-h` option.
 
-### Using export-filename
+### Using backtest-filename
 
-Normally, `backtesting-analysis` uses the latest backtest results, but if you wanted to go
-back to a previous backtest output, you need to supply the `--export-filename` option.
-You can supply the same parameter to `backtest-analysis` with the name of the final backtest
-output file. This allows you to keep historical versions of backtest results and re-analyse
-them at a later date:
+By default, `backtesting-analysis` processes the most recent backtest results. If you want to analyze results from an earlier backtest, use the `--backtest-filename` option to specify the desired file. This lets you revisit and re-analyze historical backtest outputs at any time by providing the filename of the relevant backtest result:
```

Comment:
I think the only thing missing here is stating that this will use the default `user_data/backtesting-results/` folder, and give an example where you set the `backtest-directory` but not the filename.

Reply:
Something around this?